test/oli_web/controllers/api/ingest_controller_test.exs

```diff
@@ -0,0 +1,50 @@
+defmodule OliWeb.IngestControllerTest do
```

Comment:
Just noticed that the `ingest_controller` is not under the `api` folder, so maybe we can move it out from that folder and be consistent with the app structure, wdyt?